launcher/ui/pages/instance/ModFolderPage.cpp

```diff
@@ -388,9 +390,8 @@ void ModFolderPage::on_actionInstall_mods_triggered()
     if(m_inst->typeName() != "Minecraft"){
         return; //this is a null instance or a legacy instance
     }
-    bool hasFabric = !((MinecraftInstance *)m_inst)->getPackProfile()->getComponentVersion("net.fabricmc.fabric-loader").isEmpty();
-    bool hasForge = !((MinecraftInstance *)m_inst)->getPackProfile()->getComponentVersion("net.minecraftforge").isEmpty();
-    if (!hasFabric && !hasForge) {
+    auto profile = ((MinecraftInstance *)m_inst)->getPackProfile();
+    if (profile->getModLoader() == ModAPI::Any) {
```

Comment:
This looks really non-intuitive :/
Can we add a `ModAPI::None` to the enum and use it instead? Doesn't seem like a big change

Reply:
Yeah, thought about it as well. Maybe just rename `Any` to `None` or `Unspecified`?

Reply:
Maybe `Unspecified` would be nice. Ideally we would have both `Any` and `None`, but since they mean virtually the same thing from the code's perspective, it's fine i think :)